doc/website/blog/2023-05-08-atlas-v011-kubernetes-operator.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Announcing v0.11.0: Manage database schemas with Kubernetes and Atlas"
+authors: rotemtam
+tags: [schema, migration, terraform]
+---
+:::info TL;DR
+
+You can now use the Atlas Kubernetes Operator to safely manage your database schemas with Atlas from
+within your Kubernetes cluster.
+
+[See an example](#demo-time)
+
+:::
+
+## Introduction
+
+Today, we are excited to announce the release of [Atlas v0.11.0](https://github.com/ariga/atlas/releases/tag/v0.11.0), 
+which introduces the [Atlas Kubernetes Operator](/integrations/kubernetes/operator). This release is a major 
+milestone in our mission to make Atlas the most robust and modern to manage your database schemas. With the Atlas Kubernetes
+Operator, you can now manage your database schemas with Atlas from within your Kubernetes cluster.
+
+In this release, we also introduce a new concept to Atlas - 
+["Diff Policies"](https://atlasgo.io/declarative/apply#diff-policy) - which allow you to customize the
+way Atlas plans database migrations for you. This concept is directly related to the Kubernetes Operator, and we how 
+below. 
+
+### What are Kubernetes Operators?
+
+Kubernetes has taken the cloud infrastructure world by storm mostly thanks to its declarative API. When working
+with Kubernetes, developers provide their cluster's desired configuration to the Kubernetes API, and Kubernetes
+is responsible for reconciling the actual state of the cluster with the desired state. This allows developers to
+focus on the desired state of their cluster, and let Kubernetes handle the complexities of how to get there.
+
+This works out incredibly well for stateless components, such as containers, network configuration and access
+policies. The benefit of stateless components is that they can be replaced at any time, and Kubernetes can 
+simply create a new instance of the component with the desired configuration. For stateful resources, such as databases 
+however, this is not the case. Throwing away a running database and creating a new one with the desired configuration
+is not an option.
+
+For this reason, reconciling the desired state of a database with its actual state
+can be a complex task that requires a lot of domain knowledge. [Kubernetes Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+were introduced to the Kubernetes ecosystem to help users manage complex stateful resources by codifying
+this type of domain knowledge into a Kubernetes controller.
+
+### What is the Atlas Kubernetes Operator?
+
+The Atlas Kubernetes Operator is a Kubernetes controller that uses Atlas to manage
+your database schema. The Atlas Kubernetes Operator allows you to define the desired schema
+and apply it to your database using the Kubernetes API.
+
+### Declarative schema migrations
+
+![](https://user-images.githubusercontent.com/1522681/236139615-1d10feea-8b19-46a2-905b-b614883c48c8.png)
```

Comment:
Don't forget to upload it before merging.

Reply:
It's already there